src/cascadia/TerminalSettingsModel/Profile.idl

```diff
@@ -46,6 +46,8 @@ namespace Microsoft.Terminal.Settings.Model
         void CreateUnfocusedAppearance();
         void DeleteUnfocusedAppearance();
 
+        // True if the user explicitly removed this Profile from settings.json.
+        Boolean Deleted;

```

Comment:
you can probably make this getter-only since CascadiaSettings(Serialization) uses the members directly ... right? Or do we want to expose setting this to the public API?